src/main/java/com/datastax/cdm/cql/codec/TEXT_DoubleCodec.java

```diff
@@ -50,7 +50,7 @@ public ByteBuffer encode(Double value, @NotNull ProtocolVersion protocolVersion)
     @Override
     public Double decode(ByteBuffer bytes, @NotNull ProtocolVersion protocolVersion) {
         String stringValue = TypeCodecs.TEXT.decode(bytes, protocolVersion);
-        return new Double(stringValue);
+        return Double.valueOf(stringValue);
```

Comment:
FYI - This addresses the deprecation of `new Double(String value)` in newer versions with the correct one to use.